inc/auth.class.php

```diff
@@ -127,7 +127,8 @@ static function getMenuContent() {
 
 
    /**
-    * Is the user exists in the DB
+    * Check user existence in DB
+    * @global DBmysql $DB
```

Comment:
Nice catch.
I used it to help Netbeans auto-complete which fail on $DB elsely...
I will use @var which is still active ;-)